lms/product/plugin/course_copy.py

```diff
@@ -14,11 +14,9 @@ def __init__(self, file_service: FileService):
     def is_file_in_course(self, course_id, file_id, type_) -> bool:
         """Check if `file_id` belongs to `course_id`."""
 
-        file = self._file_service.get(lms_id=file_id, type_=type_)
-        if not file or file.course_id != course_id:
-            return False
-
-        return True
+        return bool(
```

Comment:
We now directly query for the course_id so if we get a file back, we return True.